metrics/dropwizard-metrics/src/test/java/org/apache/iotdb/metrics/dropwizard/DropwizardMetricTestPlan.java

```diff
@@ -52,7 +53,6 @@ private static void test(Integer metric, Integer tag) {
   }
 
   public static void main(String[] args) {
```

Comment:
where a main() method here?

Reply:
This main method is only used for performance and memory test between dropwizard and micrometer, so whether should I need to remove it?

Reply:
This main method is only used for performance and memory test between dropwizard and micrometer, so whether should I need to remove it?